src/assets/stylesheets/shared/_custom-mixins.scss

```diff
@@ -0,0 +1,99 @@
+@import 'bootstrap/mixins';
```

Comment:
How does it find "bootstrap" and "patternfly"? My IDE can't identify the source for them.